apps/gallery/test/marionette/manifest.ini

```diff
@@ -1,3 +1,4 @@
 [edit_image_test.js]
 [fullscreen_image_test.js]
 [delete_image_test.js]
+[empty_image_test.js]
```

Comment:
I don't think we need to update manifest.ini file. See https://bugzilla.mozilla.org/show_bug.cgi?id=1072582 for bug to delete these files. https://bugzilla.mozilla.org/show_bug.cgi?id=1039140#c19 indicating that it's not used in production